change/@fluentui-react-charting-ebb094dd-b52c-43f1-b604-00481d4733c0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29113&commitId=551fc1b95c4249488c2ced69bcc795acfc893010&env=prod&clientType=FLUENTUI)


  <details><summary><b>ActivityItem</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ActivityItem.Personas - RTL.default.chromium.png|0| Added|
|ActivityItem.Personas.default.chromium.png|0| Added|
|ActivityItem.Root - RTL.default.chromium.png|0| Added|
|ActivityItem.Root.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
Screenshot change not related to this PR.